index.js

```diff
@@ -11,37 +11,69 @@ function Editable(el){
   Emitter.call(this);
   this.el = el;
   this._bind();
+  this.form = null;
+}
+
+Editable.prototype._createForm = function(initialText){
+  var self = this;
+  var input = h('input', { type: 'text', value: initialText });
+  var submit = h('input', { type: 'submit' });
+  var cancel = h('input', { type: 'reset' })
+  self.form = h('form.editable',
+    { onsubmit: onsubmit, onreset: onreset },
+    input,
+    submit,
+    cancel
+  );
+
+  function onsubmit(ev){
+    ev.preventDefault();
+    self.form.disabled = input.disabled = submit.disabled =
+    cancel.disabled = true;
+    self._publish(input.value, function(err){
+      if (!err) self.set(input.value);
+      self.form.parentNode.replaceChild(self.el, self.form);
+      self.emit('postupdate', input.value);
+      self.form = null;
+
+    });
+  }
+  function onreset(ev){
```

Comment:
can you please try to follow this project's coding style, especially

- 2 whitespace for indentation
- no empty lines before the end of functions
- no double white lines
- `} else {`